hologram_dashboard/fetcher/flags.cc

```diff
@@ -0,0 +1,24 @@
+/*
+*   Copyright 2020 Google LLC
+*
+*   Licensed under the Apache License, Version 2.0 (the "License");
+*   you may not use this file except in compliance with the License.
+*   You may obtain a copy of the License at
+*
+*       https://www.apache.org/licenses/LICENSE-2.0
+*
+*   Unless required by applicable law or agreed to in writing, software
+*   distributed under the License is distributed on an "AS IS" BASIS,
+*   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*   See the License for the specific language governing permissions and
+*   limitations under the License.
+*/
+
+#include "flags.h"
+
+DEFINE_string(chipper_batch_job_cell, "", 
+    "The job running cell of Chipper Batch.");
+DEFINE_string(chipper_gdpr_batch_job_cell, "", 
+    "The job running cell of Chipper GDPR pipeline.");
+DEFINE_string(config_file_path, "", 
```

Comment:
The flag name is too generic, make it concrete.

Reply:
what do you mean by too generic could you give me an example of a more concrete flag name?